docs/README.md

```diff
@@ -560,21 +585,21 @@ const href = await cozy.client.files.getArchiveLink(["/foo/hello.txt"], "secretp
 - `name` is the optional name for the generated archive file (default "files").
 
 
-### `cozy.client.getFilePath(file, folder)`
+### `cozy.client.files.getFilePath(file, folder)`
 
-`cozy.client.getFilePath(file, folder)` is a helper that generate the file path from root directory. It may be used to specify the path parameter for functions like
+`cozy.client.files.getFilePath(file, folder)` is a helper that generate the file path from root directory. It may be used to specify the path parameter for functions like
```

Comment:
> `cozy.client.files.getFilePath(file, folder)` is a helper that generates the file path from root directory. It may be used to specify the path parameter for functions like